README.md

```diff
@@ -20,7 +20,7 @@ MinIO-Operator brings native MinIO, [MCS](https://github.com/minio/mcs), and [KE
 
 - Kubernetes version v1.17.0 and above for compatibility. MinIO Operator uses `k8s/client-go` v0.18.0.
 - `kubectl` configured to refer to a Kubernetes cluster.
-- Create the required PVs as [explained here](https://github.com/minio/minio-operator/blob/master/docs/creating-pv-manually.md).
+- Create the required PVs using [direct CSI driver](https://github.com/minio/minio-operator/blob/master/docs/using-direct-csi.md).
```

Comment:
Actually we can have two options here e.g. 

```
Create the required PVs using
- Direct CSI Driver
- Manual PV Creation
- Auto provisioning of local PVs
```

Reply:
No I think only one way @nitisht all other methods are not recommended.